### DISCUSSION TITLE
Continue Watching / Up Next

### DISCUSSION THREAD

**Discussion by mieliespoor** [jellyfin/jellyfin-androidtv#4265] [General]
Had to switch from Plex to Jellyfin recently and one thing I noticed is that when you are watching a series, and there are new additional episodes added to the season, or you haven’t watched all episodes in the season, there is no “Continue watching” or “Up next” section showing the series titles and providing a short cut to the newly added episodes. 

Is this perhaps just on my end, or is this in fact something missing from the app?

if the latter, I can create an issue to get this request added to the backlog.

**Comment by nielsvanvelzen** [comment]
Probably something on your end. The app implements all home screen section types, including next up & continue watching.
